trajan/readers/omb.py

```diff
@@ -291,76 +299,82 @@ def read_omb_csv(path_in: Path,
                     (trajectory, obs_waves_imu, frequencies_waves_imu)),
                 attrs={
                     "_FillValue": "NaN",
+                    "standard_name": "sea_surface_wave_variance_spectral_density",
+                    "long_name": "1-dimensional wave elevation vertical energy spectrum, removing the bins dominated by low frequency noise",
                 }),
             #
             'pcutoff':
             xr.DataArray(dims=["trajectory", "obs_waves_imu"],
                          data=np.nan * np.ones((trajectory, obs_waves_imu)),
                          attrs={
                              "_FillValue": "NaN",
+                             "long_name": "Cutoff bin number for the bins dominated by low frequency noise; below this, bins are dominated by low frequency noise; determined automatically by a peak-finding algorithm",
```

Comment:
I am not sure if it is really a problem, but I believe `long_name` is supposed to be a bit shorter (e.g. this would be a bit long x/y-axis label :-). This sounds more like a `description`. There is however no standard attribute for `description`, so I guess it is file to keep it like this until we find something better.

Reply:
Yes, I think this is fine/better. But also a little unclear to me what would be the best strategy here.